hw/ip/pwm/dv/env/pwm_env.core

```diff
@@ -9,18 +9,21 @@ filesets:
     depend:
       - lowrisc:dv:ralgen
       - lowrisc:dv:cip_lib
-      - lowrisc:dv:pwm_agent
       - lowrisc:ip:pwm
     files:
+      - pwm_if.sv
```

Comment:
My apologies, I wasn't clear. The PWM monitor should be maintained in a separate fusesoc core file that can be picked up separately by the chip level env, without having to compile the whole env code too.